content/general/limits.textile

```diff
@@ -352,9 +352,24 @@ h2(#message). Message limits
 Message limits relate to the number, rate and bandwidth of "messages":/channels/messages on your account.
 
 |_=. Limit |_=. Free |_=. PAYG |_=. Business (legacy) |_=. Committed Use |
+| "Server-side batching [beta]":#message-publish-rate | 50 | 50 | 50 | 50 |
```

Comment:
I don't think we need to include any server-side batching values as limits on this page. They're not something that can currently be customised, varied per package, or really impacts usage.